File: visual_basic/integracion_con_base_de_datos/guardar_un_contacto_basico/conexion.vb
Imports MySql.Data.MySqlClient
Module conexion
    Public Function ObtenerConexion() As MySqlConnection
        Dim cadenaConexion As String = "server=localhost;user id=root;password=;database=guardar_los_contactos_basicos"
        Dim conexion As New MySqlConnection(cadenaConexion)
        Return conexion
    End Function
End Module

File: python/buscar_contacto_por_nombre/conexion.py
import mysql.connector
def conectar():
    try:
        conexion = mysql.connector.connect(
            host="localhost",
            user="root",
            password="",
            database="buscar_contacto_por_nombre"
        )
        return conexion
    except mysql.connector.Error as e:
        print("Error al conectar:", e)
        return None